### PR TITLE
Only generate manifest if it isn't already in memory

### PR DESCRIPTION
**Pull request by SleeplessOne1917**
This should hopefully address #2234

### CODE REVIEW

File: src/server/handlers/manifest-handler.ts [hunk 1]
```diff
@@ -9,15 +9,15 @@ let manifest: Awaited<ReturnType<typeof generateManifestJson>> | undefined =
   undefined;
 
 export default async (req: Request, res: Response) => {
-  if (!manifest || manifest.start_url !== getHttpBaseExternal()) {
```
Comment (SleeplessOne1917) [inline src/server/handlers/manifest-handler.ts:12]: `start_url` is hard-coded to `/`, so this if statement was always true with the right hand side of the or expression. Removing that condition should hopefully mean the manifest gets generated if it's not already stored in memory.

File: src/server/handlers/manifest-handler.ts [hunk 2]
```diff
@@ -9,15 +9,15 @@ let manifest: Awaited<ReturnType<typeof generateManifestJson>> | undefined =
...
+  if (!manifest) {
     const headers = setForwardedHeaders(req.headers);
```
Comment (dessalines) [inline src/server/handlers/manifest-handler.ts:13]: Maybe we should do away with this also? The forwardedHeaders includes auth, and might make this uncacheable.

**Review by dessalines — COMMENTED** [review]
I'm not sure what makes this get cached in memory, and when it doesn't... But feel free to merge when you think its good.

**Comment by SleeplessOne1917** [comment]
> I'm not sure what makes this get cached in memory, and when it doesn't... But feel free to merge when you think its good.

I'm not sure if "cache" is the right word. It's more that the lifetime of top-level module variables lasts until the nodejs process terminates. [This stack overflow post provides a more in-depth explanation.](https://stackoverflow.com/a/62843303)

File: src/server/handlers/manifest-handler.ts (hunk 2 above)
Comment (SleeplessOne1917) [reply src/server/handlers/manifest-handler.ts:13]: "This" being the `if` statement? The entire function?

**Pull request merged**